packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -240,7 +240,7 @@ export default function lwc(pluginOptions: RollupLwcOptions = {}): Plugin {
 
                         if (type === RegistryType.alias) {
                             // specifier must be in in namespace/name format
-                            const hasValidSpecifier = specifier.split('/').length > 1;
+                            const hasValidSpecifier = specifier.split('/').length === 2;
```

Comment:
This doesn't handle strings like `bar/` and `/foo` correct? Their length would be 2, even though they're invalid.

Reply:
Sorry about that I missed the previous comment.  I've updated the logic to account for variations of invalid alias specifiers.

Also want to note that a relative import (starting with a './' or '/' is not subject to the aliasing, as they would be using the actual file path, ie `./foo.js` which does not use an alias.

Here's the updated logic + unit tests.